### PR TITLE
Better french translation

### PR DESCRIPTION
**Pull request by Nilvus**
@TurboGit: time needs to be translated as date on those ones,. So fix this. See what's wrong with hour in this simple popup:

image

### CODE REVIEW

**Comment by TurboGit** [comment]
But then you have confusions in the collection pop-up, see:

image

That's why I have changed that.

File: po/fr.po [hunk 1]
```diff
@@ -19696,7 +19696,7 @@ msgstr "métadonnées"
 #: ../src/libs/collect.c:2979 ../src/libs/filtering.c:868
 #: ../src/libs/filtering.c:950
 msgid "times"
-msgstr "heures"
+msgstr "dates"
```
Comment (TurboGit) [inline po/fr.po:19699]: ok

File: po/fr.po [hunk 2]
```diff
@@ -5026,7 +5026,7 @@ msgstr "nom de fichier"
 
 #: ../src/common/collection.c:612 ../src/common/collection.c:667
 msgid "capture time"
-msgstr "heure de capture"
+msgstr "date de capture"
```
Comment (TurboGit) [inline po/fr.po:5029]: this one is confusion, it should be `heure` or `date/heure,` but I think just `heure` looks better.

**Comment by AlicVB** [comment]
No need to change anything, in fact this 2 properties are just here to differentiate the UI used in collections module, but are are treated equally at the end. So better just change the property exposed in the filter module, and we will be ok !

**Comment by Nilvus** [comment]
@TurboGit: as suggested by @AlicVB, property changed on filters to use capture date instead of capture time. This PR should be good now.

**Review by TurboGit — APPROVED** [review]
Thanks!

**Pull request merged**